gemini/src/main/java/com/techempower/data/EntityGroup.java

```diff
@@ -110,29 +110,29 @@ public int compare(Identifiable o1, Identifiable o2)
           return Long.compare(o1.getId(), o2.getId());
         }
       };
-      
+
   /**
    * Compare entities that implement Comparable using their built-in compareTo
-   * method.  This should only be needed until Java 1.8 gives us 
+   * method.  This should only be needed until Java 1.8 gives us
    * Comparator.naturalOrder().
    */
   enum NaturalOrderComparator implements Comparator<Comparable<Object>> {
-        INSTANCE;
-   
-        @Override
-        public int compare(Comparable<Object> c1, Comparable<Object> c2) {
-          return c1.compareTo(c2);
-        }
-      }
-  
+    INSTANCE;
+
+    @Override
+    public int compare(Comparable<Object> c1, Comparable<Object> c2) {
+      return c1.compareTo(c2);
+    }
+  }
+
   /**
    * Gets a natural-ordering Comparator implementation for a Comparable.
    */
   @SuppressWarnings("unchecked")
   public static <T extends Comparable<? super T>> Comparator<T> naturalOrder() {
       return (Comparator<T>) EntityGroup.NaturalOrderComparator.INSTANCE;
```

Comment:
We should get rid of this eventually and use `Comparator.naturalOrder()` instead.